app/scenes/root/root.js

```diff
@@ -60,38 +58,27 @@ export default class Root extends PureComponent {
 
     init = () => {
         if (this.props.logoutRequest.status === RequestStatus.SUCCESS) {
-            this.props.actions.removeStorage().then(() => {
-                setTimeout(this.loadStoreAndScene, 1000);
-            });
+            setTimeout(() => this.loadStoreAndScene(), 1000);
         } else {
-            this.loadStoreAndScene();
+            this.loadStoreAndScene(this.props.credentials);
         }
     };
 
-    loadStoreAndScene = () => {
-        this.props.actions.loadStorage().then(() => {
-            if (this.props.credentials.token && this.props.credentials.url) {
-                this.props.actions.setStoreFromLocalData(this.props.credentials).then(() => {
-                    if (this.props.loginRequest.status === RequestStatus.SUCCESS) {
-                        this.props.actions.goToLoadTeam();
-                    } else {
-                        this.selectServer();
-                    }
-                });
-            } else {
-                this.selectServer();
-            }
-        });
+    loadStoreAndScene = (credentials = {}) => {
+        if (credentials.token && credentials.url) {
+            this.props.actions.setStoreFromLocalData(credentials).then(() => {
```

Comment:
this loads the user, team and other stuff, maybe we can do this in the background if we already have the currentUser, currentTeam and the like, what do you think?